wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Period > Published card

Fixes #10902 

This add the Period Published card.
- Max 6 data rows.
- If more than 6, `View more` is added.
- If no data rows, `No data for this period` is displayed.
- Data rows:
  - No icon is displayed.
  - Selecting a row will display the post in a web view.

To test:

---
- On a site with published content, go to Stats > select a period.
- Verify the view is:
<img width="350" alt="published" src="https://user-images.githubusercontent.com/1816888/51777435-52670580-20ba-11e9-855c-3f4f1fa6c0a5.png">

---
- Select a row.
- Verify the content is displayed in a web view.


---
- On a site with _no_ published content, go to Stats > select a period.
- Verify the view is:
<img width="350" alt="no_published" src="https://user-images.githubusercontent.com/1816888/51777393-28154800-20ba-11e9-8c3a-18eeee317517.png">
